pkg/binding/model.go

```diff
@@ -41,19 +41,32 @@ func (m *model) hasDataField() bool {
 }
 
 func newModel(annotationValue string) (*model, error) {
+
+	reObf := regexp.MustCompile(`{.*?}`)
+	obfuscatedValue := reObf.ReplaceAllStringFunc(annotationValue, func(s string) string {
+		return "{" + strings.Repeat("_", len(s)-2) + "}"
+	})
+
```

Comment:
why the existing `newModel` logic does not work anymore? the jsonpath was specified in `path` in format pretty much suitable for the parser. The value found under `path` key can be moved to`model` as is, there is no need to split it anymore.

Reply:
`re := regexp.MustCompile("[=,]")` splits the annotation value `key1={path},key2=value2,...`

Now that `path` can contain `=` signs, we have to update the way the annotation is splitted.

Reply:
> Now that path can contain = signs, we have to update the way the annotation is splitted.

True. Does the regex should be then `{.+}` or even to extract the path directly: `path={.+},?` If later, we can use `FindStringIndex` function to get `path={xxx}` and extract `{xxx}` directly, and then split the rest of keys as before?